src/system.ts

```diff
@@ -81,39 +81,33 @@ const chainSource = new ChainSource({
   arweaveBaseUrl: config.ARWEAVE_URL,
 });
 
-// Attempt to read the start height and epoch block length from the contract
-let epochZeroStartHeigth = START_HEIGHT;
-let epochBlockLength = EPOCH_BLOCK_LENGTH;
-try {
-  const resp = await got
-    .get(
-      `${config.CONTRACT_CACHE_URL}/v1/contract/${config.CONTRACT_ID}/read/epoch`,
-    )
-    .json<any>();
-  epochZeroStartHeigth = +(resp?.result?.epochZeroStartHeight ?? START_HEIGHT);
-  log.info(
-    `Got epoch zero start height from contract: ${epochZeroStartHeigth}`,
-    {
-      epochZeroStartHeigth,
-    },
-  );
-  epochBlockLength = +(resp?.result?.epochBlockLength ?? EPOCH_BLOCK_LENGTH);
-  log.info(`Got epoch block length from contract: ${epochBlockLength}`, {
-    epochBlockLength,
+const networkContract = new ArIO({
+  contractTxId: config.CONTRACT_ID,
+});
+
+// Attempt to read the start height and epoch block length from the contract - default to constants if it fails
+const { epochZeroStartHeight, epochBlockLength } = await networkContract
```

Comment:
this will be useful for https://github.com/ar-io/ar-io-observer/pull/32#discussion_r1522194740 @djwhitt